clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinatorTest.java

```diff
@@ -1435,6 +1435,7 @@ public void testWakeupAfterJoinGroupReceivedExternalCompletion() throws Exceptio
         awaitFirstHeartbeat(heartbeatReceived);
     }
 
+    @Tag("flaky") // "KAFKA-18310"
```

Comment:
Ah, I was wondering how to do that because `@Flaky` wasn't doing the job.

Reply:
Maybe we should disallow "disabled" flag by import control due to quarantine mechanism.